geoips/commandline/args.py

```diff
@@ -512,14 +514,18 @@ def add_args(parser, arglist=None):
         title="Processing workflow specifications"
     )
     if arglist is None or "procflow" in arglist:
+        if legacy:
+            help_str = """Specify procflow that should be followed for this file,
+                            located in geoips*.plugins.modules.procflows.
+                            myprocflowname.myprocflowname.
+                            The procflow string should be the procflow module
+                            name (no .py)"""
```

Comment:
This formatting can cause problems sometimes since the resulting string will contain a bunch of extra spaces. I would prefer to use something like this:
```python
            help_str = (
                "Specify procflow that should be followed for this file, "
                "located in geoips*.plugins.modules.procflows. "
                "myprocflowname.myprocflowname. "
                "The procflow string should be the procflow module "
                "name (no .py)"
            )
```

Reply:
Are there other instances of this kind of string formatting? If there is only this one let's fix it here. If there are more, let's create an issue.

Reply:
Alternatively, should this go in the same YAML file as the command help information?

Reply:
I think this is the only occurrence of this. Let me check to make sure. Since this is technically an argument supported by our legacy ``run_procflow`` calls, it's required to be here. We could duplicate this information in each ``GeoipsRun<procflow>`` command class, but that seems repetitive. Maybe we should just update the yaml instructions clarifying that you don't have to specify ``--procflow <procflow_name>`` in your test scripts if you use ``geoips run <procflow_name>``?

Reply:
It might be a good idea to make an issue for this nonetheless @jsolbrig, as all arguments created in ``geoips.commandline.args:add_args`` are formatted in the ``"""text..."""`` format highlighted in the code section above.

Reply:
Okay, can you please create that issue and resolve this comment when done?

Reply:
See #638.